TestAssets/TestProjects/TestAppSimple/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
```

Comment:
Ya, I guess not.  It looks like `TestAssets\TestProjects\PortableTests\PortableApp` is the closest, but it has a bunch of  'runtimeOptions' in its project.json for some reason.